addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3497,6 +3497,15 @@ var SnippetsMenu = Widget.extend({
                     dynamicSvg.src = colorCustomizedURL.pathname + colorCustomizedURL.search;
                 });
 
+                // The iframes must be loaded when the drag-and-drop starts and not when the snippet
+                // menu is loaded.
+                const iframeEl = $toInsert[0].querySelector("iframe[data-src]");
+                if (iframeEl) {
```

Comment:
It's the same number of lines to support having two iframes in the same snippet. `querySelectorAll` instead of `querySelector` + `for` instead of `if`